uwsift/__init__.py

```diff
@@ -87,10 +87,20 @@ def find_spec(cls, fullname: str, path=None, target=None):
 if satpy_import_path is not None:
     overwrite_import("satpy", satpy_import_path)
 
-satpy_extra_readers_import_path = config.get("satpy_extra_readers_import_path", None)
-if satpy_extra_readers_import_path is not None:
-    sys.path.insert(0, satpy_extra_readers_import_path)
-    os.environ["SATPY_CONFIG_PATH"] = satpy_extra_readers_import_path
+
+# get satpy_config_path from config file
+satpy_config_path_yml = config.get("satpy_extra_config_path", None)
+# get satpy_config_path from env variable
+satpy_config_path_env = os.getenv("SATPY_CONFIG_PATH", None)
+# expand or set environment variable SATPY_CONFIG_PATH if satpy_config_path is defined in configuration file
+if satpy_config_path_yml is not None:
+    if satpy_config_path_env is not None:
+        os.environ["SATPY_CONFIG_PATH"] = satpy_config_path_env.replace('"', "") + os.pathsep + satpy_config_path_yml
```

Comment:
What is the string replacing for? Do environment variables usually have quotes in them?

Reply:
And can we simplify this whole logic by making it two if statements:

```
if satpy_config_path_yml is not None:
    satpy_config_path_env = os.getenv("SATPY_CONFIG_PATH", "")
    if satpy_config_path_env:
        satpy_config_path_env += os.pathsep
    os.environ["SATPY_CONFIG_PATH"] = satpy_config_path_env + satpy_config_path_yml
```

Also, this seems a little risky in the long run. This depends on Satpy not being imported before this code runs. That might work now, but may not be true in the future. I think it'd be more future proof to import Satpy explicitly and then do 

```python
satpy.config.set(config_path=satpy.config.get("config_path") + [satpy_config_path_yml])
```

Reply:
> What is the string replacing for? Do environment variables usually have quotes in them?

I worked with an example `set SATPY_CONFIG_PATH="C:/path/custom1;C:/path/custom2"` from [Satpy docs](https://satpy.readthedocs.io/en/stable/config.html#component-configuration) .

Reply:
Thank you for suggesting simpler and more future proof solution. I committed this version, that works , if there is not the windows quotes problem involved. 

```
# get satpy_config_path from config file
satpy_config_path_yml = config.get("satpy_extra_config_path", None)
if satpy_config_path_yml is not None:
    satpy.config.set(config_path=satpy.config.get("config_path") + [satpy_config_path_yml])
```

Reply:
OK, Satpy alone behaves the same as I wrote above. Since there haven't been any complains, I will resolve this conversation.